apps/package.json

```diff
@@ -119,6 +119,7 @@
     "messageformat": "0.2.2",
     "mkdirp": "0.5.1",
     "mochify": "2.18.0",
+    "mockfirebase": "^0.12.0",
```

Comment:
I went to do this, but after reading https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004 , I believe `^` is the correct prefix here. It does not allow changes to the leftmost non-zero number in the [major, minor, patch] tuple, so in this case we would silently adopt 0.12.1 but not 0.13.0. Please let me know if you still think this needs to be changed.